js/components/AppVersionListComponent.jsx

```diff
@@ -95,7 +95,7 @@ var AppVersionListComponent = React.createClass({
     var errorClassSet = React.addons.classSet({
       "text-danger text-center": true,
       "hidden": this.props.fetchState === States.STATE_LOADING ||
-        this.props.fetchState === States.STATE_SUCCESS
+      this.props.fetchState === States.STATE_SUCCESS
```

Comment:
This belongs to the line on top, as this is the continuation of the boolean expression, so the indentation should be there.